custom/icds_reports/migrations/sql_templates/database_views/mwcd_dashboard.sql

```diff
@@ -0,0 +1,32 @@
+DROP VIEW IF EXISTS mwcd_report CASCADE;
+CREATE VIEW mwcd_report AS
+SELECT
+"awc_location_local"."state_id" AS "state_id",
+"awc_location_local"."state_name" AS "state_name",
+"awc_location_local"."state_site_code" AS "state_site_code",
+"awc_location_local"."aggregation_level" AS "aggregation_level",
+agg_awc.month as month,
+COALESCE(agg_awc.num_launched_awcs,0) as num_launched_awcs,
+COALESCE(agg_awc.num_launched_districts,0) as num_launched_districts,
+COALESCE(agg_awc.num_launched_states,0) as num_launched_states,
+COALESCE(agg_awc.awc_with_gm_devices,0) as awc_with_gm_devices,
+COALESCE(agg_awc.cases_household,0) as cases_household,
+COALESCE(agg_awc.cases_child_health,0) as cases_child_health,
+COALESCE(agg_awc.cases_ccs_pregnant+ agg_awc.cases_ccs_lactating,0) as total_mothers,
+COALESCE(agg_ls.num_supervisor_launched,0) as num_ls_launched
+
+FROM "awc_location_local"
```

Comment:
I think other views have used `awc_location_months_local` or something similar. Are we intentionally moving to using the awc_location_local table? Do you have any stats that explain the performance difference between the view vs table?

Reply:
yeah for this view I don't think we need awc_location_months_local view. I did not find the stats on them. But I'm pretty sure the table would be faster, can pull out the stat once the current firefighting ends.